docs/user-guide/rug/generators.md

```diff
@@ -67,48 +156,32 @@ subdirectories.
 !!! note ""
     Because all of the Atomist files are hidden under the `.atomist`
     directory, our generator project is still a fully functioning,
-    perfectly valid MkDocs project.
+    perfectly valid Spring Boot project.
 
 Let's take a close look at the Rug generator script.
 
 ## A Basic Generator Script
 
 The generator script's `#!typescript populate` method is invoked after
 the model project's files have been copied to the target project.  A
-possible generator script for the MkDocs generated discussed above
-could look like this:
+possible generator script for the project discussed above could look like this:
```

Comment:
We should probably change this to read something like, "The default contents of the generator script we added above look like the following:".